knora/dsplib/models/value.py

```diff
@@ -636,7 +636,8 @@ def __init__(self,
                  iri: Optional[str] = None,
                  ark_url: Optional[str] = None,
                  vark_url: Optional[str] = None):
-        m = re.match("^(http)s?://([\\w\\.\\-~]+)?(:\\d{,6})?(/[\\w\\-~]+)*(#[\\w\\-~]*)?", str(value))
```

Comment:
The old URI validation was quite far away from what an URI actually is, and from what the API allows